pkg/controller/subscription/rhmiConfigs/main_test.go

```diff
@@ -204,7 +203,7 @@ func TestUpdateStatus(t *testing.T) {
 			expectedSchedule: &integreatlyv1alpha1.UpgradeSchedule{
 				For: time.
 					Date(now().Year(), now().Month(), now().Day(), 0, 0, 0, 0, time.UTC).
-					AddDate(0, 0, 7-int(now().Weekday())).
+					AddDate(0, 0, (7-int(now().Weekday()))%7).
```

Comment:
Is the problem that weekday returns 0-6 (0 = Sunday) but we subtract from 7? If so, should we just change it to `6-...` instead of the mod 7?

Reply:
@pb82 Then on Mondays (with the value of 1) it would add `6-1` days to the current date (so the next Saturday) instead of `7-1`? I think this would break the test entirely.

Reply:
Ah I get it now. We want to make sure it returns Sunday on any current day. Yes, your change should work.